### ISSUE TITLE
work with keycloack and flutter web? 

### ISSUE THREAD

**Issue by SpatariuRares** [dart-lang/oauth2#146]
this package work with keycloack on a web app in flutter?

**Comment by ahmednfwela** [comment]
no, it's only a dart package that implements very basic oauth flows, consider using [package:oidc](https://pub.dev/packages/oidc), which has support for all platforms.
